### DISCUSSION TITLE
Have you ever thought of shipping a default JRE in future Windows update like you did with .NET CLR ?

### DISCUSSION THREAD

**Discussion by zoraj** [microsoft/openjdk#192] [Q&A] [answered]
Doing so will be favoring Java developer adopting JavaFX as UI framework.

**Comment by karianna** [comment, marked as answer]
Hi @Zo-MM - The recommended way to bundle Java applications is to have the jlinked runtime components with your application as opposed to relying on an operating system wide JRE.
